### PR TITLE
fix: move change alias to property on edit

### PR DESCRIPTION
**Pull request by BtencateSphereon**

### CODE REVIEW

File: src/navigation/machines/oid4vciStateNavigation.tsx [hunk 1]
```diff
@@ -135,6 +135,11 @@ const navigateAddContact = async (args: OID4VCIMachineNavigationArgs): Promise<v
       name: contact.contact.displayName,
       federations: federationParties,
       uri: contact.uri,
+      identities: contact.identities,
+      description: issuerBranding?.[0].description,
```
Comment (sanderPostma) [inline src/navigation/machines/oid4vciStateNavigation.tsx]: Can't see the entire code, but can issuerBranding be an empty array?

If so maybe something like
````
const branding = issuerBranding?.[0] || {}
navigation.navigate(MainRoutesEnum.OID4VCI, {
  screen: ScreenRoutesEnum.NEW_CONTACT_ADD,
  params: {
    name: contact.contact.displayName,
    federations: federationParties,
    uri: contact.uri,
    identities: contact.identities,
    description: branding.description,
    clientUri: branding.clientUri,
    tosUri: branding.tosUri,
    policyUri: branding.policyUri
````

File: src/screens/NewContactAddScreen/index.tsx [hunk 2]
```diff
@@ -70,9 +57,31 @@ const NewContactAddScreen: FC<Props> = (props: Props): ReactElement => {
     }, [onBackPress]),
   );
 
+  useEffect((): void => {
+    if (!federations) {
+      return;
+    }
+    const branded = federations.map(federation => fetchBrandingForContact(federation));
+    Promise.all(branded).then(result => setBrandedFederations(result));
+  }, []);
```
Comment (sanderPostma) [inline src/screens/NewContactAddScreen/index.tsx:66]: Is federations updatable?
-->
```
}, [federations]);

**Review by sanderPostma — CHANGES_REQUESTED** [review]

File: src/screens/NewContactAddScreen/index.tsx (hunk 2 above)
Comment (BtencateSphereon) [reply src/screens/NewContactAddScreen/index.tsx:66]: no it is not, i need to look at this later anyways for the issue where it is untrusted before checking anyways. so will make changes then

File: src/navigation/machines/oid4vciStateNavigation.tsx (hunk 1 above)
Comment (BtencateSphereon) [reply src/navigation/machines/oid4vciStateNavigation.tsx]: fair enough, changed

**Review by sanderPostma — APPROVED** [review]

**Comment by sanderPostma** [comment]
LGTM

**Pull request merged**